docs/docs-rules/user-stories.md

```
@@ -86,6 +86,37 @@ docs/user-stories/
 
 #### 開発戦略
 
+##### 開発パターン: Parallel Change + Skeleton Pattern
+
+Clean Architectureに従った機能追加では、既存コードを壊さずに新機能を追加するため、
+**Parallel Change**パターンと**Skeleton**パターンを組み合わせて採用する。
+
+**Phase構成**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)